### PR TITLE
add Pedro Oliveira as delegate

### PR DESCRIPTION
**Pull request by pcbo**
# Please Read and check all that apply 

## general 

- [ ] no lint or test errors

## If adding delegate

In order to prevent impersonation attacks we ask that the entity to be added as a delegate make a public post either with a link to this pull request or including the same address referenced in the pull request. This post should be on a platform you have already established reputation on. So if you are active on Celo Forum post there or if you have an active Twitter account post there. 

- LINK TO PROOF 

Checklist of common mistakes

- [ ] the image is located in the public/logos directory
- [ ] the image filename has zero spaces in it
- [ ] the image path in the json exactly matches the actual file name

### CODE REVIEW

**Comment by pcbo** [comment]
https://x.com/pcbo/status/1833077864880623806

File: public/logos/delegatees/base-babe.png
Comment (pcbo) [inline public/logos/delegatees/base-babe.png:1]: i forgot to update the file name to pcbo.png 🤦

Comment (aaronmgdr) [reply public/logos/delegatees/base-babe.png:1]: can you redo ? i can fix things but not file names

Comment (pcbo) [reply public/logos/delegatees/base-babe.png:1]: i don't mind going w base-babe.png tbh, lmk if you really think i need to do a new PR

File: src/config/delegates.json [hunk 1]
```diff
@@ -1,4 +1,16 @@
 {
+  "0x3dbf55729ABcf55F3b4F7CEbfe88B7d59BE3F23F": {
+    "name": "Pedro Oliveira",
+    "address": "0x3dbf55729ABcf55F3b4F7CEbfe88B7d59BE3F23F",
+    "logoUri": "/logos/delegatees/pcbo.png",
```
Comment (aaronmgdr) [inline src/config/delegates.json:5]:
```suggestion
    "logoUri": "/logos/delegatees/base-babe.png",

**Review by aaronmgdr — APPROVED** [review]

**Pull request merged**